examples/github-minimal.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-20.04, windows-2019, macos-11]
+        os: [ubuntu-20.04, windows-2019, macos-12, macos-14]
```

Comment:
I do wonder if it looks a bit like a mistake to list both these versions here. Worth a comment?

Reply:
I think I would, since you can't read from this that one is Intel and one ARM. Why not `macOS-13`, though?

Reply:
have changed it to macos-13, along with our tests